crypto/evp/e_aes.c

```diff
@@ -390,22 +390,36 @@ static int aesni_xts_init_key(EVP_CIPHER_CTX *ctx, const unsigned char *key,
         return 1;
 
     if (key) {
+        const int bits = EVP_CIPHER_CTX_key_length(ctx) * 4;
```

Comment:
Why is this ```4``` and not ```8```?  Probably deserves a comment.

Reply:
Okay, I'll add the comment the original code should have had :)